hat/assets/js/apps/Iaso/domains/registry/components/selectedOrgUnit/InstanceTitle.tsx

```diff
@@ -0,0 +1,148 @@
+    const { formatMessage } = useSafeIntl();
+    const getEnketoUrl = useGetEnketoUrl(window.location.href, currentInstance);
+    const currentUser = useCurrentUser();
+
+    const currentInstanceId = useMemo(() => {
+        return params.submissionId || orgUnit?.reference_instances?.[0]?.id;
+    }, [params.submissionId, orgUnit]);
+
+    const instancesOptions = useMemo(() => {
+        return (instances || []).map(instance => {
+            const isReferenceInstance = orgUnit?.reference_instances?.some(
+                ref => ref.id === instance.id,
+            );
+            const label = `${instance.form_name} (${
+                isReferenceInstance
+                    ? `${formatMessage(MESSAGES.referenceInstance)} - `
+                    : ''
+            }${moment.unix(instance.created_at).format('LTS')})`;
+            return {
+                label,
+                value: instance.id,
+            };
+        });
+    }, [formatMessage, instances, orgUnit?.reference_instances]);
+
+    const handleChange = useCallback(
+        (_, submissionId) => {
+            const newParams: RegistryParams = {
+                ...params,
+                submissionId,
+            };
+            dispatch(redirectToReplace(baseUrls.registry, newParams));
+        },
+        [params, dispatch],
+    );
+    return (
+        <Grid container className={classes.paperTitle}>
+            <Grid xs={8} item>
+                <InputComponent
+                    type="select"
+                    disabled={isFetching}
+                    keyValue="instance"
+                    onChange={handleChange}
+                    value={isFetching ? undefined : currentInstanceId}
+                    label={MESSAGES.submission}
+                    options={instancesOptions}
+                    loading={isFetching}
+                    clearable={false}
+                    withMarginTop={false}
+                />
+            </Grid>
+            {currentInstance && (
+                <Grid
+                    xs={4}
+                    item
+                    container
+                    justifyContent="flex-end"
+                    alignItems="center"
+                    className={classes.paperTitleButtonContainer}
+                >
+                    <Box className={classes.paperTitleButton}>
+                        {userHasPermission(
+                            Permission.SUBMISSIONS_UPDATE,
+                            currentUser,
+                        ) && (
+                            <IconButton
+                                onClick={() => getEnketoUrl()}
+                                overrideIcon={EnketoIcon}
+                                color="secondary"
+                                iconSize="small"
+                                size="small"
+                                tooltipMessage={MESSAGES.editOnEnketo}
+                            />
+                        )}
```

Comment:
```suggestion
                    <DisplayIfUserHasPerm permissions = {[Permission.SUBMISSIONS_UPDATE]}>
                            <IconButton
                                onClick={() => getEnketoUrl()}
                                overrideIcon={EnketoIcon}
                                color="secondary"
                                iconSize="small"
                                size="small"
                                tooltipMessage={MESSAGES.editOnEnketo}
                            />
                        </DisplayIfUserHasPerm>
```
Just don't forget the import. Then you can remove `useCurrentUser`